doc/markdown/user/implementation/library/Usage.md

```diff
@@ -26,7 +26,7 @@ ALPAKA_FN_ACC auto doSomethingOnAccelerator(
 Kernel Definition
 -----------------
 
-There is no difference between the kernel entry point function and any other accelerator function in *alpaka* except that the kernel entry point is required to be a function object which has to have the accelerator as first argument.
+There is no difference between the kernel entry point function and any other accelerator function in *alpaka* except that it has to return void and the kernel entry point is required to be a function object which has to have the accelerator as first argument.
```

Comment:
Surely may fix this some time later, but I find this paragraph a little confusing. It starts with basically "A function has to be a function object which returns void", which is weird with both "function is function object" and the concept of "function object return type". The second part could be simplified as well. I suggest the following formulation:
A kernel entry point is [or is required to be, if we prefer this wording] a function object. Its function call operator is an alpaka accelerator function taking an accelerator as its first argument and returning void.